vms/registration/tests/test_functional_admin.py

```diff
@@ -271,56 +308,99 @@ def test_city_field(self):
         self.driver.find_element_by_id('id_unlisted_organization').send_keys('admin-org')
         self.driver.find_element_by_xpath('//form[1]').submit()
 
+        # verify successful registration
         self.assertNotEqual(self.driver.find_elements_by_class_name('messages'),
                 None)
         self.assertEqual(self.driver.find_element_by_class_name('messages').text,
                 'You have successfully registered!')
 
-        self.assertEqual(self.driver.current_url, self.live_server_url +
```

Comment:
@tapasweni-pathak Yes, because this is checking that on registration, we are directed back to home page. And the line above this checks if the message is being displayed properly. If registration had failed, we would still be on the registration page.